content/docs/pulumi-cloud/deployments/versus.md

```diff
@@ -14,16 +14,68 @@ aliases:
+### Stack-centric vs. Repository-centric
 
-The Deployments Platform offers a combination of Deployment Settings that describe all necessary dependencies, and Deployment Triggers that run deployments via Automation API, the REST API, and more. Stacks can be programmatically created and configured via the Settings REST API, and updated via Triggers. This unlocks platform scenarios that aren’t possible with tools that only offer `git push` workflows.
+**Traditional CI/CD:** Typically organized around repositories and branches. Workflows and configuration are usually tied to a specific repo structure or branch strategy.
 
-## Using Deployments with your Existing CI/CD System
+**Pulumi Deployments:** Organized around Pulumi stacks, which represent environments or deployment targets. Deployment Settings are configured per-stack, not per-repo or per-branch, giving you more flexibility in how you structure your infrastructure deployments.
 
-Many teams are at the stage where CI/CD is more than enough. They are happy with GitHub Actions, and wonder why they might consider migrating from GitHub Actions to Deployments. You don’t have to make that choice, as the two are better together. Teams often configure Pulumi Deployments on their stacks, and then trigger deployments via the REST API from within a GitHub Actions workflow. This gives you access to Deployments Platform tools such as Click to Deploy from the Pulumi Cloud UI, while keeping the familiarity of your existing CI/CD workflow. As Pulumi adds more built-in capabilities to the Deployments Platform like [Drift Detection](https://github.com/pulumi/pulumi-cloud-requests/issues/173), [TTL stacks](https://github.com/pulumi/pulumi-cloud-requests/issues/149), and [Ephemeral Environments](https://github.com/pulumi/pulumi-cloud-requests/issues/206), you’ll automatically accrue the productivity benefits.
+### Multi-stack Orchestration
 
-![Pulumi Deployments Platform Architecture](../deployments.png)
+**Traditional CI/CD:** Designed primarily for linear application deployments, where a single build artifact moves through environments sequentially.
+
+**Pulumi Deployments:** Built to handle complex multi-stack orchestration patterns common in infrastructure deployments:
+
+- Deploy multiple stacks to different regions in a coordinated fashion
+- Create complex dependencies between stacks
+- Handle parallel deployments across multiple environments
+- Deploy a shared infrastructure stack followed by multiple application-specific stacks
+
+### Infrastructure State Management
+
+**Traditional CI/CD:** Generally stateless, focused on building and deploying code that manages its own state elsewhere.
+
+**Pulumi Deployments:** Deeply integrated with Pulumi's state management system, understanding the nuances of infrastructure state:
+
+- Automatically manages state files with proper locking
```

Comment:
Any Pulumi Cloud deployment does this right?

Reply:
Fixed.